glam/tests/test_api.py

```diff
@@ -399,7 +399,7 @@ def test_versions_count(self, client):
         data = resp.json()
         assert len(data["response"]) == 4
         versions = sorted([d["version"] for d in data["response"]])
-        assert versions == sorted(["72", "71", "70", "69"])
+        assert versions == sorted([72, 71, 70, 69])
```

Comment:
It would be good to add a test for version 100 to verify in the tests that 100 > 99. You could tweak this one or duplicate it to add another one?